Headers/x11/XGServerWindow.h

```diff
@@ -84,8 +84,9 @@ typedef struct _gswindow_device_t {
   Display               *display;      /* Display this window is on */
   Window                ident;         /* Window handle */
   Window                root;          /* Handle of root window */
-  Window		parent;        /* Handle of parent window */
-  int                   screen;        /* Screeen this window is on */
+  Window                parent;        /* Handle of parent window */
+  int                   screen_id;     /* Screeen this window is on */
```

Comment:
Why keep the screen_id? In your code this will only ever point to the default screen.

Reply:
screen_id used in multiple places as Xlib screen for Xlib function calls - we should keep that structure element.